src/components/ui/formik/validationSchemas.ts

```diff
@@ -19,9 +19,7 @@ export const getUniqueNameValidationSchema = (excludedList: (string | undefined)
     (value) => !excludedList.includes(value),
   );
 
-export const sshPublicKeyValidationSchema = Yup.string()
-  .matches(SSH_PUBLIC_KEY_REGEX, {
-    message: 'SSH public key must consist of "ssh-rsa key [email]"',
-    excludeEmptyString: true,
-  })
-  .required('Required');
+export const sshPublicKeyValidationSchema = Yup.string().matches(SSH_PUBLIC_KEY_REGEX, {
```

Comment:
nit: is it result of prettier? Did not complain to me and I like former style more.

Reply:
yeah, prettier formats it this way